sources/api/netdog/Cargo.toml

```diff
@@ -1,13 +1,18 @@
 [package]
 name = "netdog"
-version = "0.1.0"
+version = "0.1.1"
 authors = ["Ben Cressey <bcressey@amazon.com>"]
 license = "Apache-2.0 OR MIT"
 edition = "2021"
 publish = false
 # Don't rebuild crate just because of changes to README.
 exclude = ["README.md"]
 
+[features]
+default = ["wicked"]
```

Comment:
it feels wrong to me that wicked is the default feature flag, but we're using systemd-networkd as the default version of netdog otherwise.

I see the change was made to pass the unit tests, is it possible to rewrite the tests to either work with systemd-networkd or to specifically use the wicked features?

Reply:
Good catch, I forgot to add the unit test change, I can modify the unit test to work with both, as systemd-networkd only adds extra lines (if i change the test to only verify that the first line is the expected it should pass on both and we can flip the default)